Assembler.Base/BaseFrameHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Assembler.Core;
+using Assembler.Core.Entities;
+
+namespace Assembler.Base
+{
+    public abstract class BaseFrameHandler<TFrame, TMessage> : IFrameHandler<TFrame, TMessage>
+        where TFrame : BaseFrame
+        where TMessage : BaseMessageInAssembly
+    {
+        protected readonly ITimeBasedCache<TMessage> Cache;
+        protected readonly IFactory<TFrame, string> IdentifierFactory;
+        protected readonly IMessageEnricher<TFrame, TMessage> MessageEnricher;
+        protected readonly ICreator<TMessage> MessageInAssemblyCreator;
```

Comment:
I dont like the in-consistent of naming. `Cache` by itself, `MessageEnricher` without `InAssembly` but `MessageInAssemblyCreator`. 

Stick with one. My advice - add `MessageInAssembly` to every (relevant) property

Reply:
f